gapic/batch/staging.yaml

```diff
@@ -0,0 +1,6 @@
+common:
+  batch_apis: '*'
+  batch_languages: '*'
+  api_config_pattern: ${THISDIR}/../api/artman_*.yaml
+  artman_language_yaml: ${THISDIR}/../lang/common.yaml
+  batch_output_dir: ${REPOROOT}/client-library-staging/generated
```

Comment:
So currently the batch pipeline does output everything into output_dir, just like the single language x api case. Then batch_output_dir is for an additional copy step, like the final_repo_dir in the single case. For the staging.yaml file, it is set to the staging repo, but we could have other configs like a staging_eap batch config that would point it somewhere else.

Maybe it is worth just calling it final_repo_dir in the batch config instead of batch_output_dir? But it isn't quite the same thing because final_repo_dir is actually the final directory, while batch_output_dir is extended with `<lang>/<api>` before the copy step, and no packman packaging tasks are run.